Doyeon-k-fanshawe/group2_bootcamp
# Adding Video Script

## What does the PR do?
* It adds the video script pdf

## How do you check this PR?
* Pull down the branch
*  See added pdfs

## Screenshots 
None.
